### PR TITLE
Fix issue when tests throw a different error message when AE is on

### PR DESCRIPTION
**Pull request by rene-ye**
Tests throw a different error because the column name is invalid and cannot be passed into spde.

### CODE REVIEW

File: src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java [hunk 1]
```diff
@@ -201,7 +202,8 @@ else if (lobClass == Blob.class)
                         if (!verified) {
                             // Odd CharacterStream length will throw this exception
                             if (!e.getMessage().contains(TestResource.getResource("R_badStreamLength"))) {
-                                assertTrue(e.getMessage().contains(TestResource.getResource("R_streamReadError")));
+                                assertTrue(e.getMessage().contains(TestResource.getResource("R_streamReadError"))
+                                        || e.getMessage().contains(TestResource.getResource("R_aeStreamReadError")));
```
Comment (lilgreenbird) [inline src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java:206]: should we check if AE is enabled to see which error it should be?

File: src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java [hunk 2]
```diff
@@ -41,6 +41,7 @@
 import org.junit.runner.RunWith;
 
 import com.microsoft.sqlserver.jdbc.RandomUtil;
+import com.microsoft.sqlserver.jdbc.SQLServerConnection;
```
Comment (lilgreenbird) [inline src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java:44]: why?

File: src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java (hunk 1 above)
Comment (rene-ye) [reply src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java:206]: The only really good way is to check the connection string, and that's honestly pretty hacky and I don't want to add the extra logic when the test doesn't really care if AE is on or off.

Comment (lilgreenbird) [reply src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java:205]: yeah I did that in PR 1371, we should agree to do the same thing, if we're not going to check then I'll remove that from that PR too

Comment (rene-ye) [reply src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java:205]: I'm ok with not checking, it doesn't have to be consistent persay.

Comment (lilgreenbird) [reply src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java:205]: use ds.getColumnEncryptionSetting() as per discussion

**Pull request merged**